GUI/window.py

```diff
@@ -204,14 +243,33 @@ def __init__(self, **kwargs):
                                                         pos_hint={'center_x': .5, 'center_y': 2.12},
                                                         on_release=self.clear)
         self.save_button = MDFillRoundFlatIconButton(text='Save File',
-                                                        icon='download',
-                                                        size_hint=(None, None),
-                                                        pos_hint={'center_x': .35, 'center_y': 2.12},
-                                                        on_release=self.save)
+                                                     icon='download',
+                                                     size_hint=(None, None),
+                                                     pos_hint={'center_x': .35, 'center_y': 2.12},
+                                                     on_release=self.save)
+
+        self.ascii_label_1 = Label(text='[color=000000]' + chr(int(RAM[4088], 16)) + '[/color]', pos=(-187, 29),
+                                   font_size=40, markup=True)
+        self.ascii_label_2 = Label(text='[color=000000]' + chr(int(RAM[4089], 16)) + '[/color]', pos=(-146, 29),
+                                   font_size=40, markup=True)
+        self.ascii_label_3 = Label(text='[color=000000]' + chr(int(RAM[4090], 16)) + '[/color]', pos=(-100, 29),
+                                   font_size=40, markup=True)
+        self.ascii_label_4 = Label(text='[color=000000]' + chr(int(RAM[4091], 16)) + '[/color]', pos=(-54, 29),
+                                   font_size=40, markup=True)
+        self.ascii_label_5 = Label(text='[color=000000]' + chr(int(RAM[4092], 16)) + '[/color]', pos=(-8, 29),
+                                   font_size=40, markup=True)
+        self.ascii_label_6 = Label(text='[color=000000]' + chr(int(RAM[4093], 16)) + '[/color]', pos=(38, 29),
+                                   font_size=40, markup=True)
+        self.ascii_label_7 = Label(text='[color=000000]' + chr(int(RAM[4094], 16)) + '[/color]', pos=(84, 29),
+                                   font_size=40, markup=True)
+        self.ascii_label_8 = Label(text='[color=000000]' + chr(int(RAM[4095], 16)) + '[/color]', pos=(130, 29),
+                                   font_size=40, markup=True)
+
```

Comment:
The positions need to be remodeled due to merging of the hex keyboard. Id say to place them below the keyboard; merge with master and update

Reply:
Thank you for the suggestion, the ASCII grid's position has been updated to be under the Hex Keyboard in the latest commit